Barotrauma/BarotraumaShared/SharedSource/Lua/LuaSetup.cs

```diff
@@ -349,6 +349,7 @@ public void Initialize()
 			lua.Globals["load"] = (Func<string, Table, string, DynValue>)LoadString;
 			
 			lua.Globals["LuaUserData"] = UserData.CreateStatic<LuaUserData>();
+			lua.Globals["HookMethodType"] = UserData.CreateStatic<LuaHook.HookMethodType>();
```

Comment:
`HookMethodType` is already accessible by Lua via `Hook.HookMethodType`, so there's no need to create a global static enum of it

Reply:
okay, had deleted it.